mobify/platform-scaffold
# remove remember me from sign in and register

remove remember me from sign in and register

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1440

## Changes
- remove remember me from sign in and register

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- check sign in and register form
